### ISSUE TITLE
Accuracy and IoU reported in paper

### ISSUE THREAD

**Issue by khiemkhanh98** [SimJeg/FC-DenseNet#40]
Hi SimJeg

Is the Accuracy and IoU reported in your paper taken on the Validation Set or Test Set? I'm trying to reproduce your result, and I got 0.74 IoU on val set at 0.05 train loss, while the IoU on test set IoU is only 0.6, which is a significant drop. Can you confirm for me about this?

Many thanks
